addons/web/static/tests/helpers/test_utils_dom.js

```diff
@@ -20,9 +20,9 @@ var concurrency = require('web.concurrency');
  *
  * @returns {Promise}
  */
-function returnAfterNextAnimationFrame() {
-    return new Promise(async resolve => {
-        await concurrency.delay(0);
+async function returnAfterNextAnimationFrame() {
+    await concurrency.delay(0);
+    return new Promise(resolve => {
```

Comment:
await instead of return if the return value is never important

Reply:
Done.